### PR TITLE
Feat/power menu

### PR DESCRIPTION
**Pull request by caleb-at-pieces**
adds power menu page to the docs

### CODE REVIEW

**Review by shivay-at-pieces — APPROVED** [review]

File: docs/product-highlights-and-benefits/power-menu.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Pieces Dektop App Power Menu
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/power-menu.mdx:2]: Small typo here

File: docs/product-highlights-and-benefits/power-menu.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+description: The Power Menu is a fantastic way to speed up your workflow while utilizing the Pieces Desktop Application.
+---
+
+# Perform Actions with ease
+
+![Power Menu in the Pieces Desktop Application](gif's are pending export)
+
+The Power Menu is full of useful actions that you might want to take in the Desktop Application. Since it is accessibile simply through `command + return` on MacOS or `ctrl + enter` on Linux or Windows, you can access just about anything you want without taking your hands off of the keyboard.
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/power-menu.mdx:10]: Lets also mention it's available via Cmd/ctrl+Shift+p

File: docs/product-highlights-and-benefits/power-menu.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+### Quickly Access Resources
+
+![Access resources using the Power Menu in the Pieces Desktop Application](gif's are pending export)
+
+There is a comprehensive list of many resources contained within the Power Menu. From the Power Menu you can easily access things like support, documentation, plugin marketplaces, and more.
+
+### Common Actions Surfaced Higher
+
+With the edition of the Power Menu, we also added the 'Actions Engine' into the Desktop Application. The actions engine is able to know what kind of actions you typically take and where in the Desktop App you might take them, allowing for super quick and simple access to common things you do with Pieces.
```
Comment (mack-at-pieces) [inline docs/product-highlights-and-benefits/power-menu.mdx:26]: small typo here - `addition`

**Review by mack-at-pieces — COMMENTED** [review]
Great stuff - gonna be super helpful

File: docs/product-highlights-and-benefits/power-menu.mdx (hunk 1 above)
Comment (mason-at-pieces) [reply docs/product-highlights-and-benefits/power-menu.mdx]: Fixed

**Review by mason-at-pieces — APPROVED** [review]
Solid

**Pull request merged**